src/python/bot/minimizer/js_minimizer.py

```diff
@@ -109,13 +149,18 @@ def _execute(self, data):
 
         # 2) Remove comma and right-hand-side.
         #    e.g.: f(crash(), whatever) -> f(crash())
-        if index + 1 < len(testcase.tokens):
-          hypothesis = [index, index + 1]
-          testcase.prepare_test(hypothesis)
 
-      # Keep track of where the previous line started.
-      elif token.endswith('\n'):
-        last_start_of_line_token = index + 1
+        # Find the next non whitespace token after the comma.
+        i = 1
+        whitespace = []
+        while (index + 1 < len(testcase.tokens) and
```

Comment:
this loop doesn't modify the `index` variable, is this first condition needed at all?

Reply:
I guess you meant `index + i` here